src/test/unit_tests/test_nc_health.js

```diff
@@ -693,6 +695,83 @@ mocha.describe('nsfs nc health', function() {
     });
 });
 
+
+mocha.describe('health - lifecycle', function() {
+    const Health = new NSFSHealth({ config_root, config_fs, lifecycle: true });
+    const orig_lifecycle_logs_dir = config.NC_LIFECYCLE_LOGS_DIR;
+
+    mocha.before(async () => {
+        await fs_utils.create_fresh_path(config_root);
+        config.NC_LIFECYCLE_LOGS_DIR = tmp_lifecycle_logs_dir_path;
+    });
+
+    mocha.after(async () => {
+        fs_utils.folder_delete(config_root);
+        config.NC_LIFECYCLE_LOGS_DIR = orig_lifecycle_logs_dir;
+    });
+
+    mocha.beforeEach(async () => {
+        await config_fs.create_config_json_file(JSON.stringify({ NC_LIFECYCLE_LOGS_DIR: tmp_lifecycle_logs_dir_path }));
+    });
+
+    mocha.afterEach(async () => {
+        fs_utils.folder_delete(tmp_lifecycle_logs_dir_path);
+        await config_fs.delete_config_json_file();
+    });
+
+    mocha.it('Health lifecycle - lifecycle worker never ran on host', async function() {
+        try {
+            await nb_native().fs.readdir(config_fs.fs_context, tmp_lifecycle_logs_dir_path);
+            assert.fail('tmp_lifecycle_logs_dir_path should not exist');
+        } catch (err) {
+            assert.equal(err.code, 'ENOENT');
+        }
+        const health_status = await Health.nc_nsfs_health();
+        assert_lifecycle_status(health_status, {});
+    });
+
+    mocha.it('Health lifecycle', async function() {
```

Comment:
can you give more details in this test, for example: "Health lifecycle - after 1 run of the health command in the CLI".